service-catalog/mongo.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: apps/v1
```

Comment:
can you please run https://github.com/itaysk/kubectl-neat
on all files to remove redundant lines? 🙏 thanks :)